### ISSUE TITLE
Files saved to root by default

### ISSUE THREAD

**Issue by kurobeats** [jhaddix/domain#8]
Hey there,
I know you are going off the assumption that the user is using Kali but for those who are not, can I suggest the results are saved to a local folder such as "reports"?

```
echo "set FILENAME /root/Desktop/$domain$stamp.csv" >> $domain$stamp.resource
echo "set FILENAME /root/Desktop/$domain$stamp.lst" >> $domain$stamp.resource
```

**Comment by giaurbs** [comment]
Just made a pull request to fix this. Let's see :)

**Comment by jhaddix** [comment]
jhaddix merged commit e9a4850 into jhaddix:master 3 minutes ago
